### PR TITLE
Add code to allow overriding MTU from earhly YAML.

### PR DESCRIPTION
**Pull request by dchw**
Simple as `earthly config global.cni_mtu 12345`.

### CODE REVIEW

File: cmd/earthly/main.go [hunk 1]
```diff
@@ -903,6 +903,12 @@ func (app *earthlyApp) before(context *cli.Context) error {
 	app.buildkitdSettings.AdditionalArgs = app.cfg.Global.BuildkitAdditionalArgs
 	app.buildkitdSettings.AdditionalConfig = app.cfg.Global.BuildkitAdditionalConfig
 
+	// ensure the MTU is something allowable in IPv4, cap enforced by type. Zero is autodetect.
+	if app.buildkitdSettings.CniMtu != 0 && app.buildkitdSettings.CniMtu < 68 {
+		return errors.New("Invalid overridden MTU size")
```
Comment (vladaionescu) [inline cmd/earthly/main.go:908]: Nit: lower-case errors

```suggestion
		return errors.New("invalid overridden MTU size")
```

**Review by vladaionescu — APPROVED** [review]

File: cmd/earthly/main.go (hunk 1 above)
Comment (dchw) [reply cmd/earthly/main.go]: I get that idiomatic Go wants lower case errors, but it feels wrong to present sentences that start with lower case to the user.

Comment (vladaionescu) [reply cmd/earthly/main.go]: It's actually presented as `Error: invalid overridden MTU size` to the user

Comment (dchw) [reply cmd/earthly/main.go]: Still feels wrong to me. But I am willing to follow convention.

**Review by alexcb — APPROVED** [review]

**Pull request merged**